pyrogram/client/client.py

```diff
@@ -45,14 +45,17 @@
 from pyrogram.api.types import (
     User, Chat, Channel,
     PeerUser, PeerChannel,
+    InputUser,
     InputPeerEmpty, InputPeerSelf,
-    InputPeerUser, InputPeerChat, InputPeerChannel
+    InputPeerUser, InputPeerChat, InputPeerChannel,
+    InputPhoneContact
 )
 from pyrogram.crypto import AES
 from pyrogram.session import Auth, Session
 from pyrogram.session.internals import MsgId
 from .input_media import InputMedia
 from .style import Markdown, HTML
+from typing import List, Union
```

Comment:
Unfortunately the `typing` module is only available for Python 3.5+. We should remove this in order to support 3.4